scripts/globals/events/login_campaign_data.lua

```diff
@@ -0,0 +1,463 @@
+-- This file is called by login_campaign.lua and require()'s no file, it should not be require()'d by any
```

Comment:
I think its fine to keep this file the size it is. I don't think we need to have a file that holds 3+ years of historic data.

I'd rather have one example entry with lots of options laid out so end-users can copy/paste it and modify it to their liking.

Reply:
Also, the last thing we want is a PR every month adding a new entry to this file, invalidating everything that has come before. We're interested in emulating current retail; so at best new entries will only be "valid" for a couple of weeks before fading into obscurity